docs: simple basic status, account and product switcher docs (#3255)
twilio-labs/paste: packages/paste-website/src/pages/components/status-badge/index.mdx

  title: 'Status Badge - Components',
  package: '@twilio-paste/status',
  description:

>     'A Status Badge should be used to communicate the status of a process, or the connection to an application or service.',

Reviewer: ```suggestion
    'A Status Badge communicates the status of a process or connectivity to an application or service.',
```

nitpick here because "connection to an application or service" makes it sound like the Status Badge shows the relationship to an application or service.

  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/status"
  storybookUrl="/?path=/story/components-status--"
  name="Status Badge"
  data={props.data}
/>

> 

Reviewer: chore: Change labels that say "In Progress" to "In-progress"

## About Status Badge

> The Status badge is an implementation of the the [Status Pattern](/patterns/status), pre-built for you to display and indicate the status of a process or connection in your product.
> 
> It pairs Status level with color and icon to assist with visual design and accessbility.

Reviewer: ```suggestion
Status badge is an implementation of the [Status Pattern](/patterns/status) to display the status of a process or connectivity in your product.
```

Reviewer: Removed the last line since it's repeated in the Accessibility section right below.

### Accessibility

> The idea of the Status Badge is that helps you communicate a status in a more accessible way but automatically pairing an icon with a color that represents a specific status level. This ensure there are distinct visual difference between each status type.
> 
> Care must still be taken to ensure the word used to describe the status in the badge is unique to the status level it is paired with for any given screen.

Reviewer: ```suggestion
Status Badge communicates a status in an accessible way by automatically pairing an icon with a color that represents a specific status type. This ensures there are distinct visual differences between each status type.

Care must still be taken to ensure the word used to describe the status in the badge is unique to the status type it is paired with for any given screen.
```

## Examples

> ### Variants

Reviewer: suggestion(non-blocking): Group all process types together in one example, and all connectivity types together in one example

> Status Badge labels should be one to three words, or numbers.

Reviewer: ```suggestion
Status Badge labels should be 1–3 words.
```
